tests/e2e/utils/flows.js

```diff
@@ -980,6 +982,9 @@ export const merchantWCP = {
 			'select[name="item_id"]'
 		);
 		await selectItem.click();
+		await page.click(
```

Comment:
Since the only test using this function has been skipped, is this change still necessary?

Reply:
Good point! I added it as a potential workaround for the test we've finally skipped so we no longer need this new change.

Updated in [c9dd76e](https://github.com/Automattic/woocommerce-payments/pull/10057/commits/c9dd76ebc88de0418b4a2617f480c08760d0182c).